wcivf/settings/base.py

```diff
@@ -214,6 +214,7 @@
 
 GOCARDLESS_REDIRECT_URL = "https://whocanivotefor.co.uk/donate/process/"
 
+YNR_BASE_URL = "https://candidates.democracyclub.org.uk/"
```

Comment:
I don't think we need to define this twice: https://github.com/DemocracyClub/WhoCanIVoteFor/blob/9375842052b611d81e7e5d089fcc2641c5ade977/wcivf/settings/base.py#L176

Apart from that, looks good as long as @JoeMitchell is happy with the text

Reply:
I'm happy!